ui/raidboss/data/06-ew/raid/p7s.ts

```diff
@@ -880,6 +900,50 @@ const triggerSet: TriggerSet<Data> = {
         'War\'s Harvest': '生命の繁茂【乱】',
       },
     },
+    {
+      'locale': 'cn',
+      'replaceSync': {
+        'Agdistis': '阿格狄斯提斯',
+        'Immature Io': '未成熟的伊娥',
+        'Immature Minotaur': '未成熟的弥诺陶洛斯',
+        'Immature Stymphalide': '未成熟的铁爪怪鸟',
+      },
+      'replaceText': {
+        '--chasing aoe--': '--追踪AOE--',
+        '--eggs--': '--蛋--',
+        'arrow': '箭头',
+        'close': '近',
+        'far': '远',
+        'Blades of Attis': '阿提斯的叶刃',
+        'Bough of Attis': '阿提斯的巨枝',
+        'Bronze Bellows': '突风强袭',
+        'Bullish Slash': '牛魔猛击',
+        'Bullish Swipe': '牛魔横扫',
+        'Condensed Aero II': '集聚烈风',
+        'Death\'s Harvest': 'Death\'s Harvest',
```

Comment:
If there's not a translation, can you remove the english->english translation line and mark `missingTranslations: true`? Otherwise, it would be easy to forget that these sections were not done and still had more work to do.